### DISCUSSION TITLE
Is there a way to use variables in log file name?

### DISCUSSION THREAD

**Discussion by Taps7734** [bluenviron/mediamtx#2970] [Questions]
### Question

Currently I have the log file set to 
logFile: D:/MediaMTX/log/mediamtx.log

Is it possible to use the same variables used in recorded files? So that I could make them roll over daily?  Something like

logFile: D:/MediaMTX/log/mediamtx-%Y.%m.%d.log

**Comment by aler9** [comment]
Hello, the software in charge of rotating log files is logrotate. It's available on every Linux distribution and there's also a port for Windows.

We won't add this feature natively into the server.

[a comment by an automated account is not shown]
